assets/xml/objects/object_efc_star_field.xml

```diff
@@ -1,8 +1,8 @@
 ﻿<Root>
     <File Name="object_efc_star_field" Segment="6">
-        <DList Name="object_efc_star_field_DL_000080" Offset="0x80" />
+        <DList Name="object_efc_star_field_DL_000080" Offset="0x80" /> <!-- Original name is "efc_star_field_modelT" -->
         <Texture Name="object_efc_star_field_Tex_000108" OutName="tex_000108" Format="i8" Width="16" Height="64" Offset="0x108" />
-        <DList Name="object_efc_star_field_DL_000DE0" Offset="0xDE0" />
+        <DList Name="object_efc_star_field_DL_000DE0" Offset="0xDE0" /> <!-- Original name is "demo_rock_model" -->
```

Comment:
Are these the blue rays and rocks from OOT Goddesses create hyrule cutscene? And neither DL is used only the textures??

Reply:
At the very least, that's what the rock was used for:
![image](https://user-images.githubusercontent.com/12245827/221508679-64239d53-f309-48e1-b87f-cac83541e3c0.png)

None of the textures are currently used in any decompiled actor; I'm not sure this is used at all in MM.